modules/exploits/windows/ftp/ftpshell_cli_bof.rb

```diff
@@ -0,0 +1,107 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::TcpServer
+  include Msf::Exploit::Seh
+  include Msf::Exploit::Remote::Egghunter
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'FTPShell client 6.70 (Enterprise edition) Stack Buffer Overflow',
+      'Description'    => %q{
+          This module exploits a buffer overflow in the FTPShell client 6.70 (Enterprise edition) allowing remote
+          code execution.
+        },
+      'Author'   =>
+        [
+          'r4wd3r',                                            # Original exploit author
+          'Daniel Teixeira <danieljcrteixeira[at]gmail.com>'   # MSF module author
```

Comment:
It looks like you're mentioned in about a dozen modules, and usually without an associated email address.

If you like, we can add your name to the `.mailmap` file. That way your email will be auto-populated in the documentation and when viewing the module info within `msfconsole`. Up to you.

```
# grep -rni 'Daniel Teixeira' modules/
modules/exploits/multi/http/oscommerce_installer_unauth_code_exec.rb:22:        'Daniel Teixeira'      # MSF module author
modules/exploits/windows/browser/exodus.rb:27:          'Daniel Teixeira' # MSF module author
modules/exploits/windows/ftp/ayukov_nftp.rb:23:          'Daniel Teixeira',  # MSF module author
modules/exploits/windows/ftp/labf_nfsaxe.rb:23:          'Daniel Teixeira'  # MSF module author
modules/exploits/windows/http/dupscts_bof.rb:27:          'Daniel Teixeira'  # Metasploit module
modules/exploits/windows/http/vxsrchs_bof.rb:26:          'Daniel Teixeira'
modules/exploits/windows/http/syncbreeze_bof.rb:26:          'Daniel Teixeira',
modules/exploits/windows/http/disksorter_bof.rb:26:          'Daniel Teixeira'
modules/exploits/windows/fileformat/syncbreeze_xml.rb:22:          'Daniel Teixeira'
modules/exploits/windows/fileformat/dupscout_xml.rb:22:          'Daniel Teixeira'
modules/exploits/windows/misc/cloudme_sync.rb:24:          'Daniel Teixeira' # MSF module author
modules/exploits/windows/misc/disk_savvy_adm.rb:24:          'Daniel Teixeira'
modules/auxiliary/dos/http/slowloris.py:24:        'Daniel Teixeira',  # Metasploit module (Ruby)
```

Reply:
Thank you, I would appreciate that if you could.

Reply:
#10222

Reply:
Thank you